filebeat/module/nginx/ingress_controller/test/test.log-expected.json

```diff
@@ -1017,5 +1257,69 @@
         "user_agent.os.name": "Mac OS X",
         "user_agent.os.version": "10.14",
         "user_agent.version": "72.0."
+    },
+    {
+        "@timestamp": "2020-02-07T12:02:42.000Z",
+        "event.category": [
+            "web"
+        ],
+        "event.dataset": "nginx.ingress_controller",
+        "event.kind": "event",
+        "event.module": "nginx",
+        "event.outcome": "success",
+        "event.timezone": "-02:00",
+        "event.type": [
+            "info"
+        ],
+        "fileset.name": "ingress_controller",
+        "http.request.method": "GET",
+        "http.response.body.bytes": 61,
+        "http.response.status_code": 200,
+        "http.version": "1.1",
+        "input.type": "log",
+        "log.offset": 5730,
+        "nginx.ingress_controller.http.request.id": "835136ae24486dbb4156dcbe21f5d402",
+        "nginx.ingress_controller.http.request.length": 348,
+        "nginx.ingress_controller.http.request.time": 0.001,
+        "nginx.ingress_controller.remote_ip_list": [
+            "192.168.64.14"
+        ],
+        "nginx.ingress_controller.upstream.alternative_name": "",
+        "nginx.ingress_controller.upstream.ip": "172.17.0.7",
+        "nginx.ingress_controller.upstream.name": "default-web2-8080",
+        "nginx.ingress_controller.upstream.port": 8080,
+        "nginx.ingress_controller.upstream.response.length_total": 161,
+        "nginx.ingress_controller.upstream.response.length_list": [
+            "61",
+            "100"
+        ],
+        "nginx.ingress_controller.upstream.response.status_code": 203,
```

Comment:
Just to confirm, is this the status code received by the client? I think we should report the one received by the client here.

Reply:
Yes, the logic here is to report the last value here as this is the one the client receives.

Reply:
Perfect, thanks.